plugin/plugin_test.go

```diff
@@ -203,6 +205,26 @@ func TestNewCLIPlugin_PathError(t *testing.T) {
 			t.Errorf("NewCLIPlugin() plugin = %v, want nil", p)
 		}
 	})
+
+	t.Run("plugin timeout error", func(t *testing.T) {
+		if runtime.GOOS == "windows" {
+			t.Skip("skipping test on Windows")
+		}
+		expectedErrMsg := "'sleep 10' command execution timeout: signal: killed"
+		ctxWithTimout, cancel := context.WithTimeout(ctx, 10 * time.Millisecond)
+		defer cancel()
+
+		var twoSeconds proto.Command
+		twoSeconds = "10"
```

Comment:
nit:
not sure if this should be `tenSeconds`? as the context has a 10 sec timeout above.

Reply:
So context has 10 millis timeout and the `sleep 2` has 2 second timeout to test the shellout timeout usecase.